newIDE/app/src/MainFrame/Providers.js

```diff
@@ -77,6 +80,8 @@ export default class Providers extends React.Component<Props, {||}> {
                                   children({
                                     i18n,
                                     eventsFunctionsExtensionsState,
+                                    loadPreferencesValues,
+                                    savePreferencesValues,
```

Comment:
MainFrame is not using these directly, better use a context consumer when needed inside MainFrame render

Reply:
Oh :o! that would be a better approach. I will update it :)